eslint-config-landr/index.js

```diff
@@ -61,6 +61,11 @@ module.exports = {
             {
                 enforceForRenamedProperties: false,
             },
+            {
+                VariableDeclarator: {
+                    object: true
```

Comment:
Isn't this rule `true` by default, so this won't change anything?

Reply:
i use it on the webapp, so looks like it's not true by default. check the linked PR on description that explain pb.

Reply:
If you check the link you provide, someone actually suggests to disable the rule that you've added to resolve the problem: https://github.com/eslint/eslint/issues/10782#issuecomment-415561951

Surely I'm misunderstanding something?

Reply:
looks like that you allow the error...  I use here exacly the same rule that propes in the PR : 
`/* eslint prefer-destructuring: ["error", {VariableDeclarator: {object: true}}] */`
Anyway i am closing this and keeping it on the webapp falcor as specific parameters (As people sugest)... if at somepoint we want to make it global we car reactivate this PR